### PR TITLE
Allow precompiled Kotlin settings plugins to have plugins block

### PR DESCRIPTION
**Pull request by rieske**
https://github.com/gradle/gradle/issues/15416

### CODE REVIEW

File: subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/precompile/v1/PrecompiledScriptTemplates.kt [hunk 1]
```diff
@@ -91,6 +91,21 @@ open class PrecompiledSettingsScript(
     target: Settings
 ) : DefaultKotlinScript(SettingsScriptHost(target)), PluginAware by target {
 
+    /**
+     * Configures the plugin dependencies for this project.
```
Comment (big-guy) [inline subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/precompile/v1/PrecompiledScriptTemplates.kt:95]:
```suggestion
     * Configures the plugin dependencies for this settings script.
```

**Review by big-guy — CHANGES_REQUESTED** [review]

**Review by jjohannes — APPROVED** [review]
Man, if we would have know that it is that simple. :) 

Thanks for tackling this before 7.1
Looks good :+1:

**Comment by rieske** [comment]
@bot-gradle test and merge

**Comment by rieske** [comment]
@bot-gradle test and merge

**Comment by big-guy** [comment]
@bot-gradle test and merge

**Comment by rieske** [comment]
@bot-gradle test and merge

**Comment by blindpirate** [comment]
I have fixed the verification problem on `master`.

@bot-gradle test and merge

**Pull request merged**